src/form/elements/FormSelect3.ts

```diff
@@ -60,9 +60,6 @@ export default class FormSelect3 extends AFormElement<IFormSelect3> {
     const options = Object.assign(this.elementDesc.options, {multiple: this.isMultiple});
     this.select3 = new Select3(options);
     this.$node.node().appendChild(this.select3.node);
-
-    this.elementDesc.attributes.clazz = this.elementDesc.attributes.clazz.replace('form-control', ''); // filter out the form-control class, because the border it creates doesn't contain the whole element due to absolute positioning and it isn't necessary
-    this.setAttributes(this.$node.select('.select3'), this.elementDesc.attributes);
```

Comment:
The lines where already there in the previous release, but seem to work incorrectly. In this (develop) release they are working correctly, but result in wrong element attributes. Hence, I removed these two lines to get the same DOM as in the previous release.  

Alternatively, we could replace `form-control` with `select3` to persist the CSS class.

```
this.elementDesc.attributes.clazz = this.elementDesc.attributes.clazz.replace('form-control', 'select3');
```

@lehnerchristian Please choose which version is correct in your opinion. IMO it's the version I commited.

Reply:
so no custom classes can be applied to FormSelect3? this was done through `this.setAttributes`

some time ago I had some problems with the `form-control` class and Select3 where the border was not as wide as the Select3 element itself. don't you have the same error?

Reply:
my error with the border was probably caused by another CSS rule applying an absolute positioning

but being able to set custom CSS classes would still be nice if it's necessary

Reply:
I changed the code that classes are now added using the D3 `classed` method. Hence existing CSS classes are kept.